src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java

```diff
@@ -324,38 +324,28 @@ else if (currentAst.getType() != TokenTypes.VARIABLE_DEF) {
      * Returns the distance to variable usage for in the child node.
      *
      * @param childNode child node.
-     * @param varIdent variable variable identifier.
      * @param currentDistToVarUsage current distance to the variable usage.
      * @return the distance to variable usage for in the child node.
      */
-    private static int getDistToVariableUsageInChildNode(DetailAST childNode, DetailAST varIdent,
+    private static int getDistToVariableUsageInChildNode(DetailAST childNode,
                                                          int currentDistToVarUsage) {
-        DetailAST examineNode = childNode;
-        if (examineNode.getType() == TokenTypes.LABELED_STAT) {
-            examineNode = examineNode.getFirstChild().getNextSibling();
-        }
 
         int resultDist = currentDistToVarUsage;
-        switch (examineNode.getType()) {
+
+        switch (childNode.getType()) {
             case TokenTypes.SLIST:
                 resultDist = 0;
                 break;
             case TokenTypes.LITERAL_FOR:
             case TokenTypes.LITERAL_WHILE:
             case TokenTypes.LITERAL_DO:
             case TokenTypes.LITERAL_IF:
-            case TokenTypes.LITERAL_SWITCH:
-                if (isVariableInOperatorExpr(examineNode, varIdent)) {
-                    resultDist++;
-                }
-                else {
-                    // variable usage is in inner scope
-                    // reset counters, because we can't determine distance
-                    resultDist = 0;
-                }
+                // variable usage is in inner scope, treated as 1 block
+                // or in operator expression, then distance + 1
+                resultDist++;
```

Comment:
here we were checking if the variable isn't in operator expression then it is in inner scope so we set the distance to 0 because `validateBetweenScopes` property is false.

my changes were not to set dist to zero but to add one. As we are not interested in the actual distance we just treat the whole inner scope as one block.So, even if the property is off we will get a violation to move the variable near the block. now we don't need this if condition` if(isVariableInOperatorExpr(examineNode, varIdent)) `as in both cases (inner scope and in operator expression) we add 1 to the distance.

the other removals was reported by Pitest and they are just redundant conditions and it is safe to remove like the switch case `LITERAL_SWITCH `is now being handled in the default case same for this condition

```
if (examineNode.getType() == TokenTypes.LABELED_STAT) {
            examineNode = examineNode.getFirstChild().getNextSibling();}
```